build.sbt

```diff
@@ -52,13 +59,13 @@ Revolver.settings
 // Make "package" and "assembly" depend on "scalastyle".
 lazy val styleTask = taskKey[Unit]("a task that wraps 'scalastyle' with no input parameters.")
 styleTask := { val _ = (scalastyle in Compile).toTask("").value }
-(Keys.`package` in Compile) <<= (Keys.`package` in Compile) dependsOn styleTask
+  (Keys.`package` in Compile) <<= (Keys.`package` in Compile) dependsOn styleTask
 AssemblyKeys.assembly <<= AssemblyKeys.assembly dependsOn styleTask
 
 // Make "test:test" depend on "test:scalastyle"
 lazy val testStyleTask = taskKey[Unit]("a task that wraps 'test:scalastyle' with no input parameters.")
 testStyleTask := { val _ = (scalastyle in Test).toTask("").value }
-(test in Test) <<= (test in Test) dependsOn (testStyleTask)
+  (test in Test) <<= (test in Test) dependsOn (testStyleTask)
```

Comment:
Not sure.  Emacs decided to do that when I re-indented the file.  I can un-indent them.